templates/rest_framework/api.html

```diff
@@ -21,11 +21,11 @@
           <span class="icon-bar"></span>
           <span class="icon-bar"></span>
         </button>
-        <a class="navbar-brand" href="https://archive.lcogt.net">LCOGT Science Archive</a>
+        <a class="navbar-brand" href="{{navbar_title_url}}">{{navbar_title_text}}</a>
       </div>
       <div id="navbar" class="navbar-collapse collapse">
         <ul class="nav navbar-nav">
-          <li><a href="https://lcogt.net/documentation/archive-documentation/">Documentation</a></li>
+          <li><a href="https://observatorycontrolsystem.github.io/api/science_archive/">Documentation</a></li>
```

Comment:
Hmm. We may want to make the documentation link configurable too. As it stands now, we definitely want our science-archive to point to https://lcogt.net/documentation/archive-documentation/ (rather than the auto-generated API docs), but of course other observatories won't.

Reply:
I'll add it as an environment value and set the default to the autogenerated docs. The fourth task in [this story](https://www.pivotaltracker.com/story/show/179881278) didn't mention an env setting so I didn't even think about it. But it definitely makes sense to have it configurable.